api/src/main/java/org/openmrs/module/esaudereports/reporting/library/queries/sapr/apr/CohortQueries.java

```diff
@@ -0,0 +1,97 @@
+package org.openmrs.module.esaudereports.reporting.library.queries.sapr.apr;
+
+/**
+ * Created by Hamilton Mutaquiha on 8/23/17.
+ */
+public class CohortQueries {
```

Comment:
Though this class is named in different package, we have such a class name somewhere else. If we get to a point where we need to access/create beans, we might get a class. Rename it to reflect what it is doing or migrate that code to be housed under one class with that name